release_dashboard/lib/enums/conductor_status.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Keys for the release status which is in [Map<K, V>] format.
+enum conductorStatus {
```

Comment:
types should follow UpperCamelCase

Reply:
Ditto about defining this in the same file as a widget if that is its only usage.

Reply:
This enum is used in different widgets as well. A more specific dartdoc has been added.